aws/README.md

```diff
@@ -42,6 +43,23 @@ To be documented
 To be documented
 
 
+## scan_ses_bounce_owners.py
+
+Get the suppression list from SES, and scan for resources tagged as
+owned by suppressed emails. This is useful for finding resources owned
+by people who have left Sage.
+
+### Dependencies
+
+The following python packages are required:
+
+* boto3
+
+### Usage
+
+AWS_PROFILE=organizations ./scan_ses_bounce_owners.py
```

Comment:
don't you also need to set the region, i.e. `AWS_DEFAULT_REGION=us-east-1` ?

Reply:
hm, I suppose that depends on user configuration. I have the regions set in my `.aws/config` for each profile, but otherwise yeah that variable would be needed too.